### PR TITLE
Change links to HTTP, link to our site, other cleanup

### PR DESCRIPTION
**Pull request by cassidyjames**

### CODE REVIEW

**Comment by cassidyjames** [comment]
Hold off for a sec, trying to figure out a prettier icon setup. :)

**Comment by cassidyjames** [comment]
Alright, reopening

File: README.md [hunk 1]
```diff
@@ -1,15 +1,30 @@
...
+    <img title="Chat icon" src="http://elementary.io/images/docs/human-interface-guidelines/icons/64/internet-chat.svg" />
+    <img title="Photos icon" src="http://elementary.io/images/docs/human-interface-guidelines/icons/64/multimedia-photo-manager.svg" />
+    <img title="Videos icon" src="http://elementary.io/images/docs/human-interface-guidelines/icons/64/multimedia-video-player.svg" />
+    <img title="Online Accounts icon" src="http://elementary.io/images/docs/human-interface-guidelines/icons/64/preferences-desktop-online-accounts.svg" />
+    <img title="Terminal icon" src="http://elementary.io/images/docs/human-interface-guidelines/icons/64/utilities-terminal.svg" />
+ </div>
+
 # elementary Icons
-elementary icons is an original set of vector icons designed specifically for elementary OS and its desktop environment: Pantheon.
+An original set of vector icons designed specifically for [elementary OS](http://elementary.io) and its desktop environment, Pantheon.
```
Comment (danrabbit) [inline README.md]: re-grammar that colon bruh

Comment (cassidyjames) [inline README.md]: It's an appositive, bruh. But I'll add it back. I know you love your colon.

**Pull request merged**